src/cljs/athens/views.cljs

```diff
@@ -86,7 +88,13 @@
 
          :else [:<>
                 (when @modal [filesystem/window])
-                [:div (stylefy/use-style app-wrapper-style)
+                [:div (use-style app-wrapper-style
+                                 {:class (join " "
+                                               [(cond
+                                                  (= (get-os) :windows) "os-windows"
+                                                  (= (get-os) :mac) "os-mac"
+                                                  (= (get-os) :linux) "os-linux")
+                                                (when (electron?) "is-electron")])})
```

Comment:
Need a better solution to derefing events that are defined only when `electron.cljs` is required via `:main` build, and not browser `:app` build.

Reply:
This is also going to be relevant for joining a remote graph via a link like (athens://my-db). I'm currently struggling my way through learning electron. It's clear to me that there are some problems with how we're using electron if we want to make better use of the electron APIs, but I don't know enough yet to know exactly what we need to do to make the situation better.

Reply:
@juniusfree this was the dereffing issue you said you wanted to work on. It's not exactly a reagent issue, it's more of a complexity of the codebase :/

Reply:
@tangjeff0 Thanks for clarifying. I can't do that yet. I'll look into other issues.

Reply:
Sorry @juniusfree this was actually not the derefing. https://github.com/athensresearch/athens/discussions/1099#discussioncomment-735178 is what i meant